lib/psych.rb

```diff
@@ -307,7 +307,7 @@ def self.unsafe_load yaml, filename: nil, fallback: false, symbolize_names: fals
   # A Psych::DisallowedClass exception will be raised if the yaml contains a
   # class that isn't in the +permitted_classes+ list.
   #
-  # A Psych::BadAlias exception will be raised if the yaml contains aliases
+  # A Psych::AliasesNotEnabled exception will be raised if the yaml contains aliases
   # but the +aliases+ keyword argument is set to false.
```

Comment:
This is a breaking change. Should it stay like this and wait until a major version bump, or should I make `AliasesNotEnabled` a subclass of `BadAlias` (to make the change non-breaking)?

Reply:
You should make it a subclass.  I think `BadAlias` could still be a problem in the case that there is an alias with no corresponding anchor